frontend/src/components/Navbar.vue

```diff
@@ -4,7 +4,7 @@
             <div class="relative flex items-center justify-between h-14">
 
                 <div class="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start">
-                    <div class="flex-shrink-0 flex items-center font-bold text-lg text-gray-800"><router-link to="/">Character Connect</router-link></div>
+                    <div class="flex-shrink-0 flex items-center font-bold text-lg text-gray-800"><router-link to="/"><img class="logo-max-width" src='../assets/Character_Connect_Logo.jpg'></router-link></div>
```

Comment:
<img width="766" alt="image" src="https://user-images.githubusercontent.com/52263746/156856064-85e34656-0a8a-4b37-a909-69718d9970da.png">


This is what the Navbar looks like on my laptop, I don't think this is the intended effect :D. This line might need some tweaking